test/strings/search.jl

```diff
@@ -97,6 +97,7 @@ for str in [astr]
     @test findprev('l', str, 2) == nothing
     @test findlast(',', str) == 6
     @test findprev(',', str, 5) == nothing
+    @test findlast(str, "") == nothing
```

Comment:
Might also be good to add
```jl
@test findlast(str^2, str) == nothing
```
i.e. a case where the pattern is longer than the string to be searched, but the latter is nonempty.